docs/BreakingChanges/168 DataContractJsonSerializer incompatible with ES6 (and V8).md

```diff
@@ -0,0 +1,35 @@
+## 168: DataContractJsonSerializer incompatible with ES6 (and V8)
```

Comment:
This should be expressed in terms of what's fixed, not what's broken. so:
## 168 Serialization of control characters with DataContractJsonSerialzer is now compatible with ES6 (and V8)